virttest/utils_test/libvirt.py

```diff
@@ -1499,6 +1499,31 @@ def check_state():
         return utils_misc.wait_for(check_state, timeout)
 
 
+def check_status_output(status, output='',
+                        expected_fails=[],
+                        skip_if=[],
+                        any_error=False,
+                        expected_match=[]):
+    """
+    Proxy function to call check_result for commands run in vm session.
+    :param status: Exit status (used as CmdResult.exit_status)
+    :param output: Stdout and/or stderr
+    :param expected_fails: a string or list of regex of expected stderr patterns.
+                           The check will pass if any of these patterns matches.
+    :param skip_if: a string or list of regex of expected stderr patterns. The
+                    check will raise a TestSkipError if any of these patterns matches.
+    :param any_error: Whether expect on any error message. Setting to True will
+                      will override expected_fails
+    :param expected_match: a string or list of regex of expected stdout patterns.
+                           The check will pass if any of these patterns matches.
+    """
+
+    result = process.CmdResult(stderr='',
```

Comment:
If stderr is '', I don't think the string or list in expected_fails can be searched correctly. hmm, update to 'output‘?

Reply:
Thanks for finding this, @Yingshun . Fixed.